package.json

```diff
@@ -700,6 +700,8 @@
   },
   "dependencies": {
     "@azure/event-hubs": "^1.0.8",
+    "ajv": "^6.10.0",
```

Comment:
new dependency added. We may need to refresh our third-party notice also.

Reply:
I will add it when this pull request merged into master